plugin/grails-app/controllers/grails/plugin/springsecurity/ui/RegisterController.groovy

```diff
@@ -118,79 +137,116 @@ class RegisterController extends AbstractS2UiController implements GrailsConfigu
 	}
 
 	def verifyRegistration() {
+		redirectVerifyRegistration(uiRegistrationCodeStrategy.verifyRegistration(params.t))
```

Comment:
Why was verification moved out of the controller and into the default strategy?

Reply:
It also called on line 88 of this controller when Email Validation is not required.  Previously the link in the email would have been the only way to trigger this action.